kse/src/org/kse/gui/dialogs/extensions/DExtendedKeyUsage.java

```diff
@@ -263,8 +306,7 @@ private void okPressed() {
 				&& !jcbIpSecurityUser.isSelected() && !jcbTimeStamping.isSelected() && !jcbOcspStamping.isSelected()
 				&& !jcbDocumentSigning.isSelected() && !jcbAdobePDFSigning.isSelected() && !jcbTslSigning.isSelected()
 				&& !jcbEncryptedFileSystem.isSelected() && !jcbAnyExtendedKeyUsage.isSelected()
-				)
-		{
+				&& !jcbSmartcardLogon.isSelected() && !jcbCustomExtKeyUsage.isSelected()) {
```

Comment:
Not related to custom EKUs, but SmartcardLogon wasn't taken into account when checking if there was at least one extension selected, so one couldn't select only that one.

Reply:
A new feature and a bug fix, that's great!